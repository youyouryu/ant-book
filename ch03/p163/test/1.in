8
6 3 7 9 6 4 2 1
6
2 0 4
2 4 8
1 0 4 1
2 0 4
1 4 8 1
2 4 8

