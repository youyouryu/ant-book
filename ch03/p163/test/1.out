25
13
29
17
